ONSdigital/dockers
# Update docker compose

**Changes**
- Use the jwt-test-keys that are used in testing.

**How to test**
- Follow the instructions on the readme and check the app builds.
- Check that sde-console is on the ip address exposed by docker.
